sites/all/modules/publizon/publizon.features.field_instance.inc

```diff
@@ -79,15 +113,58 @@ function publizon_field_default_field_instances() {
       'module' => 'number',
       'settings' => array(),
       'type' => 'number',
-      'weight' => 2,
+      'weight' => 0,
+    ),
+  );
+
+  // Exported field_instance: 'profile2-provider_publizon-field_phone_confirm'.
+  $field_instances['profile2-provider_publizon-field_phone_confirm'] = array(
+    'bundle' => 'provider_publizon',
+    'default_value' => NULL,
+    'deleted' => 0,
+    'description' => 'Confirm your phone number',
+    'display' => array(
+      'default' => array(
+        'label' => 'above',
+        'module' => 'number',
+        'settings' => array(
+          'decimal_separator' => '.',
+          'prefix_suffix' => 1,
+          'scale' => 0,
+          'thousand_separator' => ' ',
+        ),
+        'type' => 'number_integer',
+        'weight' => 1,
+      ),
+    ),
+    'entity_type' => 'profile2',
+    'field_name' => 'field_phone_confirm',
+    'label' => 'Phone (confirm)',
+    'required' => 0,
+    'settings' => array(
+      'max' => 99999999,
+      'min' => 10000000,
+      'prefix' => '+45',
+      'suffix' => '',
+      'user_register_form' => FALSE,
+    ),
+    'widget' => array(
+      'active' => 0,
+      'module' => 'number',
+      'settings' => array(),
+      'type' => 'number',
+      'weight' => 1,
     ),
   );
 
   // Translatables
   // Included for use with string extractors like potx.
+  t('Confirm your email address');
+  t('Confirm your phone number');
   t('Email');
+  t('Email (confirm)');
   t('Phone');
-  t('Your friendly id for using in support requests.');
```

Comment:
Not sure why this translation is removed ?

Reply:
The translated text is not a part of the features and therefore it is removed (I suppose). The text is translated in https://github.com/eReolen/base/blob/develop/sites/all/modules/reol_frontend/reol_frontend.module#L272.